.scripts/create-snapshot.config.example

```diff
@@ -0,0 +1,13 @@
+# === Server ===
```

Comment:
I don't see the configuration about the populator creating the DB. At the moment for me it's failing, as it drops the DB and I get a lot of errors:

error: Unable to create organization (UnitedNations):Authorization: unable to grant 'create' privilege: create Organization: united-nations
info: ===> organizationCreation - FULL {"durationMs":11}
info: Processing organization: SDGS – food....
info: ...creating: SDGS – food
error: Unable to create organization (SDGS – food):Authorization: unable to grant 'create' privilege: create Organization: food
info: ===> organizationCreation - FULL {"durationMs":12}
info: Processing organization: FAO of Unated Nations....
info: ...creating: FAO of Unated Nations
error: Unable to create organization (FAO of Unated Nations):Authorization: unable to grant 'create' privilege: create Organization: faq
info: ===> organizationCreation - FULL {"durationMs":11}
info: Processing organization: SDGS – energy....
info: ...creating: SDGS – energy
error: Unable to create organization (SDGS – energy):Authorization: unable to grant 'create' privilege: create Organization: energy
info: ===> organizationCreation - FULL {"durationMs":10}
info: Processing organization: SDGS – water....
info: ...creating: SDGS – water
error: Unable to create organization (SDGS – water):Authorization: unable to grant 'create' privilege: create Organization: water
info: ===> organizationCreation - FULL {"durationMs":12}
info: Processing organization: SDGS – climate....
info: ...creating: SDGS – climate
error: Unable to create organization (SDGS – climate):Authorization: unable to grant 'create' privilege: create Organization: climate
info: ===> organizationCreation - FULL {"durationMs":11}
info: Processing hub: un-sdgs....
error: Unable to update hub (un-sdgs):Authorization: unable to grant 'create' privilege: updateEcoverse: un-sdgs

I think the issue is because the ecoverse is not created.